src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +17,51 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const apr = getConfig().REACT_APP_ETH_FOX_APR
```

Comment:
will need to update this logic here to multiply by 100 once you make the config change

please search the codebase for `bnOrZero` for examples of how to do this safely